tool/CMakeLists.txt

```diff
@@ -63,6 +63,7 @@ endfunction()
 if(AWSLC_INSTALL_DIR)
   build_benchmark(awslc_bm ${AWSLC_INSTALL_DIR}/include crypto)
 
+  target_compile_options(awslc_bm PUBLIC -DAWSLC_BENCHMARK)
```

Comment:
Nit: can `OPENSSL_IS_AWSLC` be used?

Reply:
Good call out, I was thinking of that variable, but couldn't remember it.